### ISSUE TITLE
Accessibility: The NetBird Settings interface for windows cannot be viewed by screen readers.

### ISSUE THREAD

**Issue by dpy013** [netbirdio/netbird#1391]
**Is your feature request related to a problem? Please describe.**

* Currently, when using the open source screen reader [NVDA](https://www.nvaccess.org/) to operate the setup screen of netbird on windows, NVDA is unable to view the settings in the setup screen.
* 
**Describe the solution you'd like**

Is it possible for the netbird team to do some work on the accessibility of the netbird client setup interface for windows?

**Describe alternatives you've considered**
none

**Additional context**

* Reproduction steps.
First download and install [NVDA](https://www.nvaccess.org/files/nvda/releases/2023.3/nvda_2023.3.exe)
Then, find the netbird icon in the notification area toolbar. 
Open Settings and use the keyboard to navigate through the items in the Settings screen to reproduce the problem.

**Comment by mlsmaycon** [comment]
Thank you, @dpy013, for your request. It makes sense. We will evaluate our current library used for the settings menu and get back to you.

**Comment by dpy013** [comment]
Hello, everyone.
Any update on current issues please?
@mlsmaycon
